iphone/Classes/MediaModule.m

```diff
@@ -458,6 +423,28 @@ - (void)setAudioSessionMode:(NSNumber *)mode
   }
 }
 
+- (NSNumber *)audioSessionMode
```

Comment:
@vijaysingh-axway You added this API but deprecated it as well? Please clarify here.

Reply:
@hansemannn Good catch! This method was added by you in your initial change for this ticket. I think it is not needed here as audioSessionCategory is already there. Even variables AUDIO_SESSION_MODE_* are also not defined there. So removed this method. Thanks.